hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/ClientMapNearCacheTest.java

```diff
@@ -83,6 +84,9 @@
 
     protected final TestHazelcastFactory hazelcastFactory = new TestHazelcastFactory();
 
+    private static final Random rand = new Random();
```

Comment:
`Random` should not be static (and if an UPPERCASE name should be used). Would be better to make it a normal final field, so each test gets its own random generator.